core/common/src/main/java/alluxio/exception/PreconditionMessage.java

```diff
@@ -74,29 +70,14 @@
   // SEMICOLON! minimize merge conflicts by putting it on its own line
   ;
 
-  private final MessageFormat mMessage;
+  private final String mMessage;
 
   PreconditionMessage(String message) {
```

Comment:
I searched online to see if we could get the best of both world and it seems like there are three options (http://stackoverflow.com/questions/6667243/using-enum-values-as-string-literals), we originally used option 2 and now use option 1. Option 3 seems like a reasonable compromise ... thoughts?